src/gsy_e/models/strategy/energy_parameters/heatpump/pcm_tank/pcm_models.py

```diff
@@ -0,0 +1,41 @@
+from pendulum import duration
+
+
+class PCMModelBase:
+
+    def __init__(self, slot_length: duration, mass_flow_rate_kg_s: float):
+        self._mass_flow_rate = mass_flow_rate_kg_s
+        self._slot_length = slot_length
+
+    @property
+    def _soc_lut(self):
+        return {}
+
+    @property
+    def _soc_lut_key(self) -> str:
+        return ""
+
+    def get_soc(self, current_storage_temps: list):
+        return 50
+
+
+class PCMChargeModel(PCMModelBase):
+
+    @property
+    def _soc_lut(self):
+        return {}
+
+    def get_temp_after_charging(self, current_storage_temps: list, charging_temp: float) -> list:
+        return current_storage_temps
+
+
+class PCMDischargeModel(PCMModelBase):
+
+    @property
+    def _soc_lut(self):
+        return {}
```

Comment:
Why do you need to override these properties? Since the PCMModelBase already implements this property method, it will be anyways inherited to the child classes.

Reply:
Ah yes, this is only a placeholder for the code that is closed source until now. We are overwriting those for the charging and discharging. In order to not confuse the code here, I will remove the overwriting.

Reply:
Got it and thanks for the clarification!